db/schema.rb

```diff
@@ -403,6 +403,16 @@
     t.index ["email_address"], name: "index_support_users_on_email_address", unique: true
   end
 
+  create_table "tokens", force: :cascade do |t|
```

Comment:
@tijmenb keep the `tokens` table name or rename it to `authentication_tokens`?

Reply:
+1 for `authentication_tokens`